### PR TITLE
{devel,tools}[GCCcore/11.3.0] nose3 v1.3.8, coverage v7.2.7 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/nose3/nose3-1.3.8-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name = 'nose3'
+version = '1.3.8'
+
+homepage = 'https://nose.readthedocs.io/'
+description = """Nose extends unittest to make testing easier."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+
```
Comment (verdurin) [inline easybuild/easyconfigs/n/nose3/nose3-1.3.8-GCCcore-11.3.0.eb:12]: I would also remove this unnecessary blank line.

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4004.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/24837ee8e0dd0b1fdf98a65d689bc2aa for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**